applications/matter_weather_station/certification/PICS/enpoint0/Access Control cluster.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="utf-8"?><clusterPICS xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="Generic-PICS-XML-Schema.xsd"><!--
+ Autogenerated xml file 
```

Comment:
Are we sure we can include this in the repo, and not, for example, only on the FTP?
Also: can we avoid spaces in file names?

Reply:
> Are we sure we can include this in the repo, and not, for example, only on the FTP?

I don't know, let me consult it.

> Also: can we avoid spaces in file names?

I think not, those files are then used by CSA tool called PICS tool and those names may be important. @doublemis1 can you comment on that?